soc/renesas/ra/ra8m1/linker.ld

```diff
@@ -0,0 +1,608 @@
+/*
```

Comment:
Please rewrite with `zephyr_linker_sources`
https://github.com/zephyrproject-rtos/zephyr/blob/c23132afdf33dc474bc814d4ccd576e76ee6618c/soc/st/stm32/stm32h7x/CMakeLists.txt#L9

and 

`zephyr,memory-region`

https://github.com/zephyrproject-rtos/zephyr/blob/c23132afdf33dc474bc814d4ccd576e76ee6618c/dts/arm/nxp/nxp_rt11xx_cm4.dtsi#L30-L34

You probably don't need to create a new linker.ld.